test/runtime/samples/attribute-casing-unknown-namespace/_config.js

```diff
@@ -0,0 +1,18 @@
+// Test support for unknown namespaces preserving attribute case (eg svelte-native).
+
+export default {
+	html: `
```

Comment:
can you rebase against master and make sure `npm run lint` passes? we just updated our eslint config to allow only single quotes

Reply:
done